src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
+
+public abstract class Bot {
```

Comment:
Comments or JavaDoc on this class would be great. Make it as obvious as possible to people extending this class.